capabilities.go

```diff
@@ -139,5 +151,54 @@ func (c *Client) GetCapabilities(target string, port int) (response *Capabilitie
 		err = fmt.Errorf("missing %s version", DefaultServiceName)
 	}
 
+	// Parse PIKE capability
+	if pike, ok := response.Capabilities["935478af7bf2"].(map[string]interface{}); ok {
+		pikeCap := &PikeCapability{
+			Invite:  pike["invite"].(string),
+			Outputs: pike["outputs"].(string),
```

Comment:
Assuming that, the capabilities response can be from external source, I think we should check if those fields actually are in the `map` and they are strings. 

Just for safety reasons. Because, in current implementation, a `panic` can occur.
Or it's not necessary 🙈 - prove me wrong :D